### DISCUSSION TITLE
Is Trivy tool can generate SBOMs

### DISCUSSION THREAD

**Discussion by MohammedAziz02** [aquasecurity/trivy#7931] [Q&A] [answered]
### Question

can we generate SBOM with Trivy for a filesystem project (maven, python, ...), is yes How please ?

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

### Operating System

mac Darwin

### Version

```bash
0.57.0
```

**Comment by simar7** [comment, marked as answer]
https://aquasecurity.github.io/trivy/v0.57/docs/supply-chain/sbom/
